docs/docs-beta/docs/guides/external-systems/apis.md

```diff
@@ -1,4 +1,63 @@
 ---
 title: Connecting to APIs
 sidebar_position: 20
----
\ No newline at end of file
+---
+
+This guide describes how to connect to and interact with APIs in dagster. In this guide you will use dagster Resources to connect to an external API. Using a Resource allows you to standardize how you connect to an external API across your project and use configuration to customize your connections.
```

Comment:
I think this is getting close, but I'd lead with why this is important over what the guide contains / what Dagster does, for example:

> When building a data pipeline, you'll likely need to connect to a number of external APIs, each with its own specific configuration and behavior. This guide demonstrates to standardize your API connections and customize their configuration.  

(This could be better written, but hopefully it gives you an idea!)

Reply:
putting what your wrote in for now, and i'll think some more on how to phrase this!